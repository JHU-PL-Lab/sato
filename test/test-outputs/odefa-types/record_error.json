[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "r" ],
          "err_match_val": "{one=o, zero=z}",
          "err_match_expected": "{two}",
          "err_match_actual": "{one, zero}"
        }
      ]
    ],
    "err_input_seq": [],
    "err_location": "s = r.two",
    "err_steps": 72
  }
]