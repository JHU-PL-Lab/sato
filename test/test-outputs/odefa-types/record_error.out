Type errors for:
- Input sequence  : []
- Found at clause : s = r.two
--------------------
* Value    : r = {one=o, zero=z}
* Expected : {two}
* Actual   : {one, zero}
1 errors found. No further control flows exist.
